docs/project/contribution_tools.md

```diff
@@ -104,6 +104,9 @@ bazel test //...:all
 > export PATH="${HOME}/.brew/bin:${PATH}"
 > # For `llvm`:
 > export PATH="$(brew --prefix llvm)/bin:${PATH}"
+> export LDFLAGS="-L/usr/local/opt/llvm/lib"
+> export CPPFLAGS="-I/usr/local/opt/llvm/include"
+> export CC=$(which clang)
```

Comment:
```suggestion
```

Setting LDFLAGS/CPPFLAGS shouldn't be required by carbon, and the paths would need to be relative to the install path even if included.

Regarding CC, while it can be used, it shouldn't be necessary unless the user is already setting CC in some other way (which could lead to a conflict regardless). Carbon's toolchain config will internally do the `which clang` if `CC` is unset.

Overall I suggest omitting these in order to avoid creating issues.